tests/helper/helper_generic.go

```diff
@@ -312,12 +317,48 @@ func CommonBeforeEach() CommonVar {
 
 // CommonAfterEach is common function that cleans up after every test Spec (It)
 func CommonAfterEach(commonVar CommonVar) {
+	// Get details, including test result for each test spec and adds it to local testResults.txt file
+	// Ginkgo test related variables.
+	commonVar.testFileName = strings.Replace(CurrentGinkgoTestDescription().FileName[strings.LastIndex(CurrentGinkgoTestDescription().FileName, "/")+1:strings.LastIndex(CurrentGinkgoTestDescription().FileName, ".")], "_", "-", -1) + ".go"
+	commonVar.testCase = CurrentGinkgoTestDescription().FullTestText
+	commonVar.testFailed = CurrentGinkgoTestDescription().Failed
+	commonVar.testDuration = CurrentGinkgoTestDescription().Duration.Seconds()
+
+	var prNum string
+	var resultsRow string
+	prNum = os.Getenv("GIT_PR_NUMBER")
+	passedOrFailed := "PASSED"
+	if commonVar.testFailed {
+		passedOrFailed = "FAILED"
+	}
+	clusterType := "OCP"
+	if IsKubernetesCluster() {
+		clusterType = "KUBERNETES"
+	}
+	testDate := strings.Split(time.Now().Format(time.RFC3339), "T")[0]
+	resultsRow = prNum + "," + testDate + "," + clusterType + "," + commonVar.testFileName + "," + commonVar.testCase + "," + passedOrFailed + "," + strconv.FormatFloat(commonVar.testDuration, 'E', -1, 64) + "\n"
+	testResultsFile := filepath.Join("/", "tmp", "testResults.txt")
+
+	f, err := os.OpenFile(testResultsFile, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0600)
+	if err != nil {
+		fmt.Println("Error: ", err)
+	} else {
+		f.Close()
```

Comment:
If you close the file, you will not be able to write to it, please remove this.

Reply:
It's a wide practice to use `defer f.Close()` so that the file will be closed as the last thing before go runtime is done with this function.

Reply:
that f.close was left behind, removed.